src/content-helper/common/utils/constants.ts

```diff
@@ -90,20 +90,32 @@ export function getPeriodDescription(
 	/* eslint-disable @wordpress/valid-sprintf */
 	switch ( timeUnit ) {
 		case 'm':
+			if ( timeValue === 1 ) {
```

Comment:
Since those conditions are simple, I'd convert them into Yoda conditions. Also, TIL that [WP Coding Standards ](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#yoda-conditions) actually recommend to avoid Yoda conditions for for `<`, `>`, `<=` and `>=`.